app/views/personaldetails/InternationalAddressChoiceView.scala.html

```diff
@@ -21,7 +21,7 @@
 @import controllers.auth.requests.UserRequest
 @import util.LocalPartialRetriever
 @import uk.gov.hmrc.renderer.TemplateRenderer
-
+@this(form: FormWithCSRF, input: InputRadioGroup)
```

Comment:
Can this be renamed to: `inputRadioGroup: InputRadioGroup` just got me confused a sec there